com.github.dynobo.normcap.yml

```diff
@@ -0,0 +1,95 @@
+app-id: com.github.dynobo.normcap
+runtime: org.kde.Platform
+runtime-version: 6.2
+sdk: org.kde.Sdk
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
so no wayland support?

Reply:
The app should work on Wayland, too. Thanks!

Reply:
then 
```suggestion
  - --socket=fallback-x11
```

Reply:
Done.